package.json

```diff
@@ -46,7 +46,7 @@
     "chai": "^4",
     "chai-as-promised": "^8",
     "eslint": "^9",
-    "eslint-config-brightspace": "^1.2.1",
+    "eslint-config-brightspace": "^2",
     "eslint-plugin-html": "^8",
     "eslint-plugin-import": "^2",
     "eslint-plugin-sort-class-members": "^1",
```

Comment:
None of these have been needed for a while I don't think... might be worth cleaning up when we see them.